<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleWindow</class>
 <widget class="QDialog" name="BattleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Морской бой</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                    stop:0 #0a2463, stop:1 #3e92cc);
    border: 3px solid #1e3d6b;
}

QLabel {
    color: #f8f8ff;
    font-family: &quot;Arial Rounded MT Bold&quot;;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
}

QLabel#labelPlayer, QLabel#labelEnemy {
    font-size: 22px;
    font-weight: bold;
    color: #f8f8ff;
    background-color: rgba(30, 61, 107, 0.7);
    border-radius: 8px;
    padding: 5px 10px;
    border: 2px solid #d4af37;
}

QLabel#labelPlayerShips, QLabel#labelEnemyShips {
    font-size: 14px;
    color: #f8f8ff;
    font-weight: bold;
}

QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #1e3d6b, stop:1 #0a2463);
    border: 2px solid #d4af37;
    border-radius: 8px;
    padding: 8px 16px;
    font-family: &quot;Arial Rounded MT Bold&quot;;
    font-size: 14px;
    color: #f8f8ff;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #3a5f8c, stop:1 #1e3d6b);
}

QPushButton:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #0a2463, stop:1 #1e3d6b);
}

QPushButton#btnReplay {
    min-width: 120px;
    min-height: 40px;
    font-size: 16px;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #4CAF50, stop:1 #2E7D32);
    border: 2px solid #81C784;
}

QPushButton#btnReplay:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #66BB6A, stop:1 #388E3C);
}

QPushButton#btnReplay:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #2E7D32, stop:1 #1B5E20);
}

QPushButton#btnExitToMainMenu {
    min-width: 180px;
    min-height: 40px;
    font-size: 16px;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #c62828, stop:1 #b71c1c);
    border: 2px solid #e57373;
}

QPushButton#btnExitToMainMenu:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #ef5350, stop:1 #e53935);
}

QPushButton#btnExitToMainMenu:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #b71c1c, stop:1 #7f0000);
}

/* Стили для игровых клеток */
QPushButton {
    background-color: lightblue;
    border: 1px solid #1e3d6b;
    min-width: 30px;
    min-height: 30px;
}

QPushButton[hasShip=&quot;true&quot;] {
    background-color: #2c3e50;
}

QPushButton[hasShip=&quot;false&quot;] {
    background-color: lightblue;
}

QPushButton[hasShip=&quot;true&quot;]:disabled {
    background-color: #c62828;
}

QPushButton[hasShip=&quot;false&quot;]:disabled {
    background-color: white;
}
   </string>
  </property>
  <widget class="QLabel" name="labelPlayer">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ваше поле</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnemy">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Поле противника</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutPlayer" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutEnemy" native="true">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReplay">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>540</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сыграть снова</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExitToMainMenu">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>540</y>
     <width>216</width>
     <height>60</height>
    </rect>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Выход в главное меню</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayerShips">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ваши корабли: 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnemyShips">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Корабли противника: 10</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
